.github/workflows/doc-build.yml

```
@@ -1,95 +1,69 @@
-            run:
-                shell: bash -el {0}
-        steps:
-        -   name: Source caching
-            uses: actions/cache@v3
-            with:
-                path: .git
-                key: source-cache-${{ runner.os }}-${{ github.run_id }}
-                restore-keys: |
-                    source-cache-${{ runner.os }}
-        -   name: Checkout fmralign
-            uses: actions/checkout@v3
-            with:
-          # If pull request, checkout HEAD commit with all commit history
-                ref: ${{ github.event.pull_request.head.sha }}
-                fetch-depth: 0
-        -   name: Complete checkout
-            run: |
-                set -x
-                if ! git remote -v | grep upstream; then
-                  git remote add upstream https://github.com/parietal-INRIA/fmralign.git
-                fi
-                git fetch upstream
-        -   name: Merge with upstream
-            run: |
-                set -x
-                echo $(git log -1 --pretty=%B) | tee gitlog.txt
-                echo "gitlog.txt = $(cat gitlog.txt)"
-                echo $GITHUB_REF_NAME | tee merge.txt
-                if [ "$GITHUB_REF_NAME" != "main" ]; then
-                    echo "Merging $(cat merge.txt)";
-                    git pull --ff-only upstream "refs/pull/$(cat merge.txt)";
-                fi
-      # Set up environment
-        -   name: Install apt packages
-            run: |
-                sudo -E apt-get -yq update
-                sudo -E apt-get -yq --no-install-suggests --no-install-recommends install \
-                    dvipng texlive-latex-base texlive-latex-extra
-        -   name: Setup conda
-            uses: conda-incubator/setup-miniconda@v2
-            with:
-                auto-activate-base: true
-                activate-environment: ''
-                miniconda-version: latest
-                channels: conda-forge
-        -   name: Install packages in conda env
-            run: |
-                conda init bash
-                echo "conda version = $(conda --version)"
-                conda create -n testenv
-                conda install -n testenv -yq python=3.9
-                source activate testenv
-                python -m pip install --user --upgrade --progress-bar off pip
-                # See pyproject.toml for dependency group options
-                python -m pip install .[jax,test,doc]
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the repository
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Step 2: Set up Python
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      # Step 3: Install dependencies
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install torch --index-url https://download.pytorch.org/whl/cpu
+          pip install -e ".[doc]"
```

Currently failing as one of the examples calls OTT :

```suggestion
          pip install -e ".[doc, jax, test]"
```

---

Good catch!